repository/src/main/java/tech/ydb/yoj/repository/db/EntityDescriptor.java

```diff
@@ -0,0 +1,15 @@
+package tech.ydb.yoj.repository.db;
+
+import tech.ydb.yoj.databind.schema.Schema;
+
+import javax.annotation.Nullable;
+
+public record EntityDescriptor<E extends Entity<E>>(Class<E> clazz, @Nullable String tableName) {
```

Comment:
- Please mark this record as `@ExperimentalApi(issue="https://github.com/ydb-platform/yoj-project/issues/32")`.
- Remember to add `@NonNull` lombok annotation to `Class<E> clazz` record component to ensure an `EntityDescriptor` cannot be constructed with `clazz == null`
- In the future we *might* start using a more generic record, like `EntityDescriptor(EntitySchema<E> schema, String tableName)` which allows to have multiple `EntitySchema`s for the same entity class (mostly for sophisticated testing purposes).

Reply:
1 & 2 done